packages-next/auth/src/index.ts

```diff
@@ -4,14 +4,41 @@ import {
   KeystoneConfig,
   SerializedFieldMeta,
 } from '@keystone-spike/types';
-import { getExtendGraphQLSchema } from './getExtendGraphQLSchema';
-import { AuthConfig, Auth, ResolvedAuthGqlNames } from './types';
 import url from 'url';
+
+import { getExtendGraphQLSchema } from './getExtendGraphQLSchema';
 import { initFirstItemSchemaExtension } from './initFirstItemSchemaExtension';
+import { AuthConfig, Auth, ResolvedAuthGqlNames } from './types';
 
+import { signinTemplate } from './templates/signin';
+import { initTemplate } from './templates/init';
+
+/**
+ * createAuth function
```

Comment:
I don't get this bit(and the similar ones in the other comments)? If someone hovers over this, they'll see that it's a function named createAuth

Reply:
tbh I was just trying to create some visual order to the file, when I got there it was really hard to see where the different things were. happy to do this any other way (whether that's better comments, or just using since `/*` so they don't show up as TS docs)